core/src/main/java/com/klaytn/caver/methods/response/Transaction.java

```diff
@@ -339,6 +343,63 @@ public String getKey() {
         public void setKey(String key) {
             this.key = key;
         }
+
+        /**
+         * Convert TransactionData to Caver's transaction instance.
+         * @param klay The Klay instance to fill gasPrice, chainId and nonce fields when signing a transaction.
+         * @param txObject The TransactionData instance.
+         * @return AbstractTransaction.
+         */
+        public AbstractTransaction convertToCaverTransaction(Klay klay, TransactionData txObject) {
```

Comment:
Is there any reason this is implemented as a public method of TransactionData?
And the second parameter is the required value?